Extensions/xkit_patches.js

```diff
@@ -2606,3 +2601,54 @@ XKit.toast = {
 		}, 5000);
 	}
 };
+
+// Cheat the headers we send
+XKit.tools.Nx_XHR = function(params) {
+	params.timestamp = new Date().getTime();
```

Comment:
Multiple requests can theoretically be sent in the same millisecond so it would be good to add a Math.random() just in case.